### DISCUSSION TITLE
Lightweight Cookiecutter Project

### DISCUSSION THREAD

**Discussion by vik-y** [cookiecutter/cookiecutter-django#4556] [Ideas]
## New lightweight cookiecutter template

Not sure if this is the right place to ask for feedback - but - I am creating a new cookiecutter template which is supposed to be a lightweight template for creating django api based projects quickly. 

I have just started the project. It draws a lot of motivation from this project and I wanted to get feedback of the community on this idea.  https://github.com/vik-y/cookiecutter-django-lite/tree/main

## Rationale
Sometimes we need to quickly build a simple API and use all the goodness offered by django - using this template seems to be an overkill for that simple purpose because it has too much to offer. Personally for those simple use cases I often switch to fastapi and then start missing out on all the goodness that django offers. 

A lite version of this template in my opinion can be super useful for the community in context of the scenario I provided. 

Let me know your thoughts or a place where we can discuss this. Thanks.

**Comment by vik-y** [comment]
Thanks team for accepting this discussion. Looking for some good feedback and opinions.

**Comment by browniebroke** [comment]
Very nice! Very simple template which I'm sure fits a lot of use cases. 

I see you have k8s support, which is a feature that has been brought up here.
